public/js/admin.js

```diff
@@ -446,13 +448,13 @@ var deactivateUser = function(id) {
             success: function(data) {
                 displayAccountsTable();
                 const msg = ' has been&nbsp;<u><b>deactivated</b></u>&nbsp;';
-                warningSnackbar(id + ' account' + msg);
+                warningSnackbar(username + ' account' + msg);
```

Comment:
why dont we put the message in directly?

Reply:
will do